frame/balances/src/lib.rs

```diff
@@ -433,7 +433,7 @@ decl_module! {
 		///     check that the transfer will not kill the origin account.
 		///
 		/// # </weight>
-		#[weight = SimpleDispatchInfo::FixedNormal(1_000_000)]
+		#[weight = T::DbWeight::get() * (1, 1) + 200_000_000]
```

Comment:
I think the `<weight>` annotation is not up to date. Would be good to fix those as well, so we can use balances as the example of how this should be done for the rest of the modules.